kickoff/static/suggestions.js

```diff
@@ -255,45 +255,60 @@ function getElmoUrl(productName, version) {
     return url;
 }
 
-function getPreviousBuildL10nUrl(productName, version, previousBuildNumber) {
+function _getL10nChangesetsOptsOfNewVersion(productName, version, branchName, revision) {
+    var opts = {};
+    if (productName === 'thunderbird' || version < '59') {
```

Comment:
OMG, this won't work if we get up to version 590 before deprecating ship-it !!1!11!  😄

Reply:
Hehehe! Joke aside this will bite us earlier than Firefox 590. '100' is smaller than '59'. Thank for pointing this out. I'll get a fix.